php/libraries/Module.class.inc

```diff
@@ -228,7 +228,8 @@ abstract class Module extends \LORIS\Router\PrefixRouter
      *
      * @param string $page The name of the page within the module to load.
      *
-     * @return a NDB_Page subclass representing this page.
+     * @return NDB_Page|NDB_Form|NDB_Menu|NDB_Menu_Filter|NDB_Menu_Filter_Form
```

Comment:
```suggestion
     * @return NDB_Page
```

All of the things listed are subclasses of NDB_Page. If phpstan was complaining, it's probably because of the "a".

Reply:
I'll try